src/main/java/org/sonarqube/gradle/SonarPropertyComputer.java

```diff
@@ -244,7 +272,17 @@ private static void computeScanAllProperties(Project project, Map<String, Object
       .sorted()
       .collect(Collectors.toList());
 
-    properties.put(ScanProperties.PROJECT_SOURCE_DIRS, SonarUtils.joinAsCsv(mergedSources));
+    properties.put(sourcePropertyToUpdate, SonarUtils.joinAsCsv(mergedSources));
+  }
+
+  public enum InputFileType {
+    MAIN, TEST
+  }
```

Comment:
Can we use the types from sonar-plugin-api in this part of the code?

Reply:
InputFile.Type is not available